library/general/imagedigests/example.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: opa
+  namespace: digest-required
+spec:
+  containers:
+    - name: opa
+      image: openpolicyagent/opa@sha256:04ff8fce2afd1a3bc26260348e5b290e8d945b1fad4b4c16d22834c2f3a1814a
```

Comment:
thanks for the PR! Can you make the default example be deny for testing purposes? and add image with SHA in comments?
similar to this: https://github.com/open-policy-agent/gatekeeper/blob/master/library/pod-security-policy/seccomp/example.yaml#L7

Reply:
Thanks, I've updated the example to just use a tag, so it'll be denied. The digest is in the comment.